greedy_algorithms/dijkstra.cpp

```diff
@@ -90,35 +112,34 @@ void Dijkstra(Graph graph, int src) {
     print(mdist, V);
```

Comment:
would it be better for this function to return a value that can be used for testing?

Reply:
I agree. Do you think I should file an issue? Maybe I can tackle that at some other time.

Reply:
If you can incorporate this, that would be awesome and avoid further changes to this file and we can focus on other files in the repo